src/graphorm/index.ts

```diff
@@ -1535,6 +1536,34 @@ const obj = new GraphORM({
           childColumn: 'opportunityId',
         },
       },
+      questions: {
+        relation: {
+          isMany: true,
+          parentColumn: 'id',
+          childColumn: 'opportunityId',
+          customRelation: (_, parentAlias, childAlias, qb): QueryBuilder =>
+            qb
+              .where(`${childAlias}."opportunityId" = "${parentAlias}".id`)
+              .andWhere(`${childAlias}."type" = :screeningType`, {
+                screeningType: QuestionType.Screening,
+              })
+              .orderBy(`${childAlias}."questionOrder"`, 'ASC'),
+        },
+      },
+      feedbackQuestions: {
```

Comment:
not sure if these are needed if you just add the where type to the additionalQuery in OpportunityFeedbackQuestion and OpportunityScreeningQuestion